apis/nucleus/src/components/listbox/interactions/keyboard-navigation/keyboard-nav-methods.js

```diff
@@ -7,8 +9,45 @@ export const focusSearch = (container) => {
 };
 
 export const focusRow = (container) => {
-  const row = container?.querySelector('.value.last-focused, .value.selector, .value');
-  row?.setAttribute('tabIndex', 0);
-  row?.focus();
-  return row;
+  const lastFocusedRow = container?.querySelector('.value.last-focused');
+  const selectorRow = container?.querySelector('.value.selector');
+  const row = container?.querySelector('.value');
+  const rowToFocus = lastFocusedRow || selectorRow || row;
+  rowToFocus?.setAttribute('tabIndex', 0);
+  rowToFocus?.focus();
+  removeLastFocused(container);
+  return rowToFocus;
+};
+
+export const focusCyclicButton = (container) => {
+  const button = container?.querySelector('.listbox-cyclic-button:not(:disabled)');
+  button?.setAttribute('tabIndex', 0);
+  button?.focus();
+  return button;
+};
+
+export const blur = (event, keyboard) => {
```

Comment:
with the use of blur outisde kayboard-nav-container
I think it need a

```
if (!keyboard.enabled) {
   return;
}
```

or something similar
(not 100% sure yet)

as it is it break the nebula.js mashup case (when keyboard.enabled === false)

Reply:
Fixed